api/setting.go

```diff
@@ -54,10 +54,9 @@ var (
 	// configured with on startup. These values can be adjusted using the
 	// settings API.
 	DefaultPricePinSettings = PricePinSettings{
-		Enabled:          false,
-		Currency:         "usd",
-		ForexEndpointURL: "https://api.siascan.com/exchange-rate/siacoin",
-		Threshold:        0.05,
+		Enabled:   false,
```

Comment:
Having `Enabled` is redundant now it seems. If the `Explorer` is disabled, price update are disabled. If the explorer is enabled but no price is pinned that is also fine since it's a no-op.